### ISSUE TITLE
scaling messages

### ISSUE THREAD

**Issue by significance** [fairDataSociety/fds.js#35]

**Comment by plur9** [comment]
What's this about? Is it still relevant, @significance?

**Comment by significance** [comment]
yes, at the moment we just have simple json manifest of messages, and also we are pulling from many different feeds. there's a bunch than can be done here, probably should be the subject of a FDIP but we can open the discussion here. @tfius and I have kicked around a few ideas, @jmozah i'm sure has some input too.

**Comment by plur9** [comment]
@crtahlin can we label this for "roadmap sessions"?

**Comment by crtahlin** [comment]
@significance please check if this is included somewhere in the product plan. and if it isn't, should it be somewhere?

**Comment by tfius** [comment]
this is connected to #66
